taxonomies/food_groups.txt

```diff
@@ -0,0 +1,269 @@
+# The food groups taxonomy is a hierarchy with up to 3 levels.
+# Food groups are used by researchers (in particular the EREN team that created the Nutri-Score) for studies.
+# Each product can be in only 1 of each food group at each level.
+
+# The groups for each product are determined from its categories, thanks to properties in the categories taxonomy.
```

Comment:
Maybe we should tell that this taxonomy entries do not need to be translated (or do they ?)

Reply:
My plan is to make food groups much more visible than PNNS groups, so it would be a good thing to have them translated eventually.